main.py

```diff
@@ -210,6 +216,23 @@ def parse_args():
         action="store_true",
         help="Don't run generation but benchmark groundtruth (useful for debugging)",
     )
+    parser.add_argument(
+        "--one_shot",
```

Comment:
can we name this something more narrowly scopped for nuggets? Same thought for `prompt_quality` and `add_context` unless there's a more general application for these params. I think the help descriptions are nice but the cli args themselves seem a but unclear in terms of naming condition @meher-m

Reply:
Yea, great suggestion! I will update this.